lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1147,3 +1261,18 @@ def convert_to_utf8(input)
     end
   end
 end
+
+module Google
```

Comment:
nvm, just checked the Ruby doc, `dup` is intended for a shallow copy, so is `clone`. We would have to override it this way.

http://ruby-doc.org/core-2.4.0/Object.html#method-i-dup